PART_1_SHARE_INFO.md

```diff
@@ -106,7 +106,7 @@ BUILD SUCCESSFUL in 3s
 To run the tests try
 
 ```bash
-./gradlew test
+./gradlew check
```

Comment:
I tried to change `./gradlew test` to `./gradlew check` everywhere, as `check` will include the Checkstyle checks as well as the tests.